### PR TITLE
test: Update coverage report

### PR DESCRIPTION
**Pull request by ShyamOOAI**
## Summary by CodeRabbit

- **New Features**
	- Enhanced `LogsPage` with improved date and time selection, timezone management, and clipboard operations.
	- Introduced `ManagementPage` for UI testing, allowing logo updates and management functionalities.
	- Added new tests for time zone changes and streamlined test scenarios on the logs page.
	- Introduced a suite of tests for joining logs, validating various join scenarios.

- **Bug Fixes**
	- Updated confirm button interaction in `ReportsPage` to ensure visibility before clicking.

- **Documentation**
	- Improved clarity and purpose of test scenarios through restructuring and simplification.

- **Chores**
	- Added `logoManagement.spec.js` and `join.spec.js` to the Playwright UI integration tests for expanded coverage.

### CODE REVIEW

**Review by neha00290 — APPROVED** [review]

File: tests/ui-testing/playwright-tests/logoManagement.spec.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ 
+    // console.log ('URL Opened')
+ 
+    await loginPage.gotoLoginPage();
+ 
+    await loginPage.login(process.env["ZO_ROOT_USER_EMAIL"], process.env["ZO_ROOT_USER_PASSWORD"]);
+ 
+    // Step 2: Navigate to Logs Page
+    //await page.waitForTimeout(4000);  // Wait for login process
+    await page.waitForTimeout(4000);
```
[the remark this answers is not in the document]
Comment (ShyamOOAI) [reply tests/ui-testing/playwright-tests/logoManagement.spec.js:34]: updated

File: tests/ui-testing/playwright-tests/logoManagement.spec.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    await managementPage.navigateToManagement();
+    await managementPage.updateCustomLogoText("logo Auto");
+
+    console.log(`Uploading file from path: ${filePath}`);
+    //
+    await managementPage.uploadLogo(filePath);
+
+
+    await page.waitForTimeout(5000); // Adjust or remove as needed
+});
```
[the remark this answers is not in the document]
Comment (ShyamOOAI) [reply tests/ui-testing/playwright-tests/logoManagement.spec.js:46]: have added as required

File: tests/ui-testing/playwright-tests/logoManagement.spec.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+// Implementation test.spec.js
+
+import { LoginPage } from '../pages/loginPage.js';
+
+// File: tests/fileUploadTest.spec.js
+
+
+const path = require('path');
+
```
[the remark this answers is not in the document]
Comment (ShyamOOAI) [reply tests/ui-testing/playwright-tests/logoManagement.spec.js:14]: have updated

File: tests/ui-testing/pages/managementPage .js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        this.confirmDeleteButton = page.locator('[data-test="logs-search-bar-confirm-dialog-ok-btn"]');
+        this.fileUploadInput = page.locator('input[data-test="setting_ent_custom_logo_img_file_upload"]');
+    }
+
+    async navigateToManagement() {
+        await this.page.waitForSelector("[name ='home']");
+        await this.homeIcon.hover();
+        await this.page.waitForSelector('[data-test="menu-link-/settings/-item"]');
+        await this.managementMenuItem.click({ force: true });
+    }
```
[the remark this answers is not in the document]
Comment (ShyamOOAI) [reply tests/ui-testing/pages/managementPage .js:26]: both same

File: tests/ui-testing/pages/managementPage .js [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+        this.homeIcon = page.locator("[name ='home']");
+        //this.managementMenuItem = page.getByText('Management');
+        this.managementMenuItem = page.locator('[data-test="menu-link-/settings/-item"]');
+        this.submitButton = page.locator("[type='submit']").nth(1);
+        this.customLogoText = page.locator("[aria-label ='Custom Logo Text']");
+        this.saveButton = page.locator('[data-test="settings_ent_logo_custom_text_save_btn"]');
+        this.deleteLogoButton = page.locator('[data-test="setting_ent_custom_logo_img_delete_btn"]');
+        this.confirmDeleteButton = page.locator('[data-test="logs-search-bar-confirm-dialog-ok-btn"]');
+        this.fileUploadInput = page.locator('input[data-test="setting_ent_custom_logo_img_file_upload"]');
```
[the remark this answers is not in the document]
Comment (ShyamOOAI) [reply tests/ui-testing/pages/managementPage .js:18]: Updated as per appliction locator

File: tests/ui-testing/pages/managementPage .js [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+    async updateCustomLogoText(text) {
+        //  await this.page.waitForSelector("[type='submit']").nth(1);
+        await this.submitButton.click({ force: true });
+        await this.customLogoText.fill(text);
+        await this.page.waitForSelector('[data-test="settings_ent_logo_custom_text_save_btn"]');
+        await this.saveButton.click({ force: true });
+        await this.page.waitForTimeout(5000);
+
+    }
```
[the remark this answers is not in the document]
Comment (ShyamOOAI) [reply tests/ui-testing/pages/managementPage .js]: updated

File: tests/ui-testing/playwright-tests/secondsPrecisionAdded.spec.js [hunk 7]
```diff
@@ -1,446 +1,141 @@
...
-
   // Set the Date and Time Range
   await logsPage.setDateTime();
-  await logsPage.fillTimeRange(startTimeValue, endTimeValue);
-
-  // Verify the time range is displayed correctly
-  await logsPage.verifyDateTime(startDateTimeValue, endDateTimeValue);
-
+  const { browser } = test.info();
+  await logsPage.fillTimeRange(browser);
```
[the remark this answers is not in the document]
Comment (ShyamOOAI) [reply tests/ui-testing/playwright-tests/secondsPrecisionAdded.spec.js]: Have removed

**Pull request merged**